std/lua/_std/haxe/Rest.hx

```diff
@@ -0,0 +1,71 @@
+package haxe;
+import lua.Lua.select;
+import lua.TableTools.maxn;
+import lua.TableTools.pack;
+import lua.Table;
+
```

Comment:
Spaces -> Tabs again, to be consistent with the stdlib.

Reply:
Or just run haxe-formatter over it, there's other formatting issues too. :D

Reply:
FWIW I added haxe-formatter to this vim plugin : https://github.com/Chiel92/vim-autoformat/pull/297

I have it set to autorun when I open a haxe file now.